profiles/default/linux/x86/23.0/i486/musl/package.use.mask

```diff
@@ -1,8 +1,12 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License, v2
 
 # Anthony G. Basile <blueness@gentoo.org> (2020-09-06)
 # The patch added for 5.39-r1 breaks on musl i686
 # file-5.39-seccomp_sandbox.patch is okay
 # file-5.39-portage_sandbox.patch is broken
 sys-apps/file seccomp
+
+# Andreas K. Hüttel <dilfridge@gentoo.org> (2024-03-20)
```

Comment:
Masks should go on the top of a file, not the bottom. Also, put your own header there (not dilfridge's).

Reply:
> Masks should go on the top of a file, not the bottom. Also, put your own header there (not dilfridge's).

That's fine, the only reason I kept dilfridge's was that he was the original person to set this just in the wrong location to help users. 

If you want that changed to mine still that's fine I just want to clarity etiquette before doing that.